api/api.go

```diff
@@ -207,6 +207,7 @@ func New(requiredUserAgent string, requiredPassword string, cs modules.Consensus
 		router.GET("/renter", api.renterHandlerGET)
 		router.POST("/renter", RequirePassword(api.renterHandlerPOST, requiredPassword))
 		router.GET("/renter/contracts", api.renterContractsHandler)
+		router.GET("/renter/contract/*contractid", api.renterContractDetailHandler)
```

Comment:
Honestly no idea re the middleware - do you want me to make it `contracts`? Haven't used golang much so this'll be a bit of an adventure.

@DavidVorick Makes perfect sense - I'm happy to put the time in to get this merged.